Framework/DataHandling/src/LoadRawHelper.cpp

```diff
@@ -127,26 +130,40 @@ void LoadRawHelper::skipData(FILE *file, int64_t hist) {
 /// @param file :: the file pointer
 /// @param from_file :: unknown
 void LoadRawHelper::ioRaw(FILE *file, bool from_file) {
-  isisRaw->ioRAW(file, from_file);
+  isisRaw().ioRAW(file, from_file);
+}
+int LoadRawHelper::getNumberofTimeRegimes() {
+  return isisRaw().daep.n_tr_shift;
 }
-int LoadRawHelper::getNumberofTimeRegimes() { return isisRaw->daep.n_tr_shift; }
 
-void LoadRawHelper::reset() { isisRaw.reset(); }
+/**
+ * Lazy-instantiation of the ISISRAW reader
+ * @return A reference to the internal ISISRAW reader instance.
+ */
+ISISRAW2 &LoadRawHelper::isisRaw() const {
+  if (!m_isis_raw) {
+    m_isis_raw.reset(new ISISRAW2);
```

Comment:
This is extremely nit picking, but could we use `make_unique` from the kernel instead. 

This would ensure the operation is exception safe and prevents leaks
Additionally when we (eventually) cull new / delete this would save us one more to remove

Reply:
Not nit picky at all. Thanks for picking this up.